samples/sample-kotlin-android/src/main/kotlin/com/rudderstack/android/sampleapp/analytics/RudderAnalyticsUtils.kt

```diff
@@ -29,20 +29,21 @@ object RudderAnalyticsUtils {
         //wen add work manager support to this instance
         _rudderAnalytics = getInstance(
             writeKey = WRITE_KEY,
+            application = application,
+            jsonAdapter = GsonAdapter(),
             initializationListener = { success, message ->
                 listener?.onAnalyticsInitialized(WRITE_KEY, success, message)
             },
-            configuration = ConfigurationAndroid(
-                application = application,
-                GsonAdapter(),
-                dataPlaneUrl = DATA_PLANE_URL,
-                controlPlaneUrl = CONTROL_PLANE_URL,
-                trackLifecycleEvents = true,
-                recordScreenViews = true,
-                isPeriodicFlushEnabled = true,
-                logLevel = RudderLogger.LogLevel.DEBUG,
-                autoCollectAdvertId = true,
-            )
+            configurationScope = {
+                dataPlaneUrl = DATA_PLANE_URL
+                controlPlaneUrl = CONTROL_PLANE_URL
+                trackLifecycleEvents = true
+                recordScreenViews = true
+                isPeriodicFlushEnabled = true
+                autoCollectAdvertisingId = true
+                trackAutoSession = true
+                logLevel = RudderLogger.LogLevel.DEBUG
+            }
```

Comment:
Can you please revert this change for now?

I tried to disable the `lifecycle` flag to `false` and after that also I can see the lifecycle events in our dashboard.